src/components/GradeBookHeader.tsx

```diff
@@ -53,8 +57,16 @@ function GradeBookHeader({
 		<Header>
 			<CourseTitle>{name}</CourseTitle>
 			<Body>
-				<CurrentGrade label="Current Grade" grade="" />
-				<NeededGrade label="Needed Grade" grade="" />
+				<CurrentGrade
+					label="Current Grade"
+					grade={currentGradeCalculator(currentCourse).toFixed(2)}
+				/>
+				<NeededGrade
+					label="Needed Grade"
+					grade={averageGradeNeededCalculator(currentCourse).toFixed(
+						2
```

Comment:
Suuuper minor, but I'm wondering if there could be a better way to format this instead of having a lone 2? If this was done automatically, please ignore this comment! 🐤

Reply:
I am guessing this was automatic because the build passed. This is kind of where prettier kills me because I agree this doesn't look quite right, but I don't really know how to change the rules such that these odd situations don't happen.